<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpioClient</class>
 <widget class="QMainWindow" name="GpioClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GpioClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="gpio22">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(131, 131, 131);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>GPIO22</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="gpio27">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(131, 131, 131);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>GPIO27</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="gpio17">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(131, 131, 131);</string>
      </property>
      <property name="text">
       <string>GPIO17</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
